apps/studio/components/interfaces/Linter/Linter.utils.tsx

```diff
@@ -161,13 +161,23 @@ export const lintInfoMap: LintInfo[] = [
   {
     name: 'materialized_view_in_api',
     title: 'Materialized View in API',
-    icon: <Eye className="text-foreground-muted" size={15} strokeWidth={1.5} />,
+    icon: <Table2 className="text-foreground-muted" size={15} strokeWidth={1.5} />,
```

Comment:
Are these icons the wrong way around now? Feel like the view should be the eye and the table should be the table? 😅

Reply:
^ yeah agree, should be consistent with the table editor?

Reply:
i'll make the change in this PR so that we can get this through, can update if we got this one wrong thereafter